resource/ui/ts/models/log.ts

```diff
@@ -0,0 +1,149 @@
+// source: models/log.ts
+/// <reference path="../models/proto.ts" />
+/// <reference path="../typings/d3/d3.d.ts" />
+/// <reference path="../typings/mithriljs/mithril.d.ts" />
+/// <reference path="../util/chainprop.ts" />
+/// <reference path="../util/format.ts" />
+/// <reference path="../util/querycache.ts" />
+// Author: Bram Gruneir (bram+code@cockroachlabs.com)
+
+/**
+ * Models contains data models pulled from cockroach.
+ */
+module Models {
+    /**
+     * Log package represents the logs collected by each Cockroach node.
+     */
+    export module Log {
+        import promise = _mithril.MithrilPromise;
+
+        export interface LogResponseSet {
+            d: Proto.LogEntry[]
+        }
+
+        // TODO(bram): Move these into css classes.
+        var tableStyle = "border-collapse:collapse; border - spacing:0; border - color:#ccc";
+        var thStyle = "font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#efefef;text-align:center";
+        var tdStyleOddFirst = "font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#efefef;text-align:center";
+        var tdStyleOdd = "font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#f9f9f9;text-align:center";
+        var tdStyleEvenFirst = "font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#efefef;text-align:center";
+        var tdStyleEven = "font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#ccc;color:#333;background-color:#fff;text-align:center";
+
+        export class Entries {
```

Comment:
Scratch that.  I can't figure out how to get this to work.  Since it's in the constructor, I can't get access to **this** in the super call, which I need for the **this._url()**.